frontends/web/src/routes/account/account.tsx

```diff
@@ -84,17 +84,20 @@ export function Account({
   }, [code]);
 
   const maybeCheckBitsuranceStatus = useCallback(async () => {
-    if (account?.bitsuranceId) {
+    if (account?.bitsuranceStatus) {
       const insuredAccounts = await bitsuranceLookup(code);
       if (!insuredAccounts.success) {
         alertUser(insuredAccounts.errorMessage || t('genericError'));
         return;
       }
-      if (!insuredAccounts.accountCodes.includes(code)) {
+      let bitsuranceAccount = insuredAccounts.bitsuranceAccounts[0];
+      let active = bitsuranceAccount.status === 'active';
```

Comment:
I'd always prefer `const` if the intention is that the variable never changes after assignment.

Reply:
done :+1: